analyzing_merchant_data.ipynb

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
Omg all this cell blocks makes it hard to review the PRs in github.

Reply:
I know, it's really painful.

Reply:
Yeah, it's rough. Later on, we probably want to split some stuff out into helper functions stored in actual `.py` files.